moveit_ros/benchmarks/include/moveit/benchmarks/BenchmarkExecutor.h

```diff
@@ -191,36 +193,35 @@ class BenchmarkExecutor
   bool loadQueries(const std::string& regex, const std::string& scene_name, std::vector<BenchmarkRequest>& queries);
 
   /// Duplicate the given benchmark request for all combinations of start states and path constraints
-  void createRequestCombinations(const BenchmarkRequest& brequest, const std::vector<StartState>& start_states,
+  void createRequestCombinations(const BenchmarkRequest& benchmark_request, const std::vector<StartState>& start_states,
                                  const std::vector<PathConstraints>& path_constraints,
                                  std::vector<BenchmarkRequest>& combos);
 
   /// Execute the given motion plan request on the set of planners for the set number of runs
-  void runBenchmark(moveit_msgs::msg::MotionPlanRequest request,
-                    const std::map<std::string, std::vector<std::string>>& planners, int runs);
+  void runBenchmark(moveit_msgs::msg::MotionPlanRequest request);
 
-  planning_scene_monitor::PlanningSceneMonitor* psm_;
-  moveit_warehouse::PlanningSceneStorage* pss_;
-  moveit_warehouse::PlanningSceneWorldStorage* psws_;
-  moveit_warehouse::RobotStateStorage* rs_;
-  moveit_warehouse::ConstraintsStorage* cs_;
-  moveit_warehouse::TrajectoryConstraintsStorage* tcs_;
+  planning_scene_monitor::PlanningSceneMonitor* planning_scene_monitor_;
```

Comment:
Can we change these to smart pointers, so that we don't have to manage lifetime?